lib/RemoveVariableEventBehaviour.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+var inherits = require('inherits');
+
+var CommandInterceptor = require('diagram-js/lib/command/CommandInterceptor').default;
+var is = require('bpmn-js/lib/util/ModelUtil').is;
+var getBusinessObject = require('bpmn-js/lib/util/ModelUtil').getBusinessObject;
+
+/**
+ * Remove 'camunda:variableEvents' property when a startEvent is moved out of an event subProcess
+ */
+function RemoveVariableEventBehaviour(
+    modeling, injector, bpmnFactory, moddleCopy
+) {
+
+  injector.invoke(CommandInterceptor, this);
+
+  this.postExecuted(['shape.move'], function(context) {
+
+    var oldParent = context.oldParent;
+    var oldParentBusinessObject = getBusinessObject(oldParent);
+    var newParent = context.newParent;
+    var shape = context.shape;
+    var shapeBusinessObject = getBusinessObject(shape);
+    var eventDefinitions, definitionsCopy;
+
+    // if shape is a startEvent
+    if (is(shape, 'bpmn:StartEvent')) {
+
+      eventDefinitions = shapeBusinessObject.get('eventDefinitions');
+      definitionsCopy = eventDefinitions.slice();
+
+      definitionsCopy.forEach(function(def,i) {
+
+        var newDef = bpmnFactory.create('bpmn:ConditionalEventDefinition');
+        moddleCopy.copyElement(def, newDef);
+        newDef.$parent = def.$parent;
```

Comment:
We make a copy for each event definition which seems to be unnecessary when `camunda:variableEvents` is not found. Also, there are more types of event definitions, e.g. `bpmn:TimerEventDefinition`. We don't want to look into them or replace them.